Kenan-Modpack/PKs_Rebalancing/monsters/doom.json

```diff
@@ -584,7 +577,7 @@
         "description": "The arachnotron grimaces and fires its carbine!"
       }
     ],
-    "death_function": [ "SMOKEBURST", "BROKEN" ],
+    "death_function": { "effect": { "id": "death_smokeburst", "hit_self": true }, "message": "A %s explodes!" },
```

Comment:
Should be { "effect": { "id": "death_smokeburst", "hit_self": true }, "message": "A %s explodes!", "corpse_type": "BROKEN" },